include/field_follow_poke.h

```diff
@@ -20,21 +20,21 @@ u8 sub_02069EC0(LocalMapObject *mapObject);
 u8 sub_02069ED4(LocalMapObject *mapObject);
 void FollowPokeMapObjectSetParams(LocalMapObject *mapObject, int species, u8 form, BOOL shiny);
 void sub_02069F0C(LocalMapObject *mapObject, int species, u8 form, BOOL shiny, int a4);
-void FollowPokeFsysParamSet(FieldSystem *fsys, int species, u8 form, BOOL shiny, u8 gender);
+void FollowPokeFieldSystemParamSet(FieldSystem *fieldSystem, int species, u8 form, BOOL shiny, u8 gender);
 u8 GetFollowPokeSizeParamBySpecies(int species);
 int FollowPokeObj_GetSpecies(LocalMapObject *mapObject);
-BOOL FollowingPokemon_IsActive(FieldSystem *fsys);
-BOOL sub_02069FB0(FieldSystem *fsys);
-BOOL GetFollowPokePermission(FieldSystem *fsys);
+BOOL FollowingPokemon_IsActive(FieldSystem *fieldSystem);
+BOOL sub_02069FB0(FieldSystem *fieldSystem);
+BOOL GetFollowPokePermission(FieldSystem *fieldSystem);
 BOOL GetFollowPokePermissionBySpeciesAndMap(int species, u32 mapno);
 void sub_0206A040(LocalMapObject *mapObject, BOOL enable_bit);
-void sub_0206A054(FieldSystem *fsys);
-struct FieldSystemUnk108 *FsysUnkSub108_Alloc(HeapID heapId);
-void FsysUnkSub108_AddMonMood(struct FieldSystemUnk108 *unk, s8 by);
-void FsysUnkSub108_SetMonMood(struct FieldSystemUnk108 *unk, s8 mood);
-s8 FsysUnkSub108_GetMonMood(struct FieldSystemUnk108 *unk);
-void FsysUnkSub108_Set(struct FieldSystemUnk108 *a0, Pokemon *mon, u16 species, u32 personality);
-void FsysUnkSub108_MoveMoodTowardsNeutral(struct FieldSystemUnk108 *a0);
+void sub_0206A054(FieldSystem *fieldSystem);
+struct FieldSystemUnk108 *FieldSystemUnkSub108_Alloc(HeapID heapId);
+void FieldSystemUnkSub108_AddMonMood(struct FieldSystemUnk108 *unk, s8 by);
+void FieldSystemUnkSub108_SetMonMood(struct FieldSystemUnk108 *unk, s8 mood);
```

Comment:
Any idea what FieldSystemUnkSub108 is at this point?

Reply:
nope